pkg/ebpf/state.go

```diff
@@ -399,6 +414,7 @@ func (ns *networkState) GetStats(closedPollLost, closedPollReceived, tracerSkipp
 			"unordered_conns":              ns.telemetry.unorderedConns,
 			"closed_conn_dropped":          ns.telemetry.closedConnDropped,
 			"conn_dropped":                 ns.telemetry.connDropped,
+			"simultaneous_conns":           ns.telemetry.simultaneousConns,
```

Comment:
Do we need to do anything extra to expose this via expvar?

Reply:
it should be automatic, the `expvarStats` function collects everything under telemetry: https://github.com/DataDog/datadog-agent/blob/master/pkg/ebpf/tracer.go#L165